torchtext/models/roberta/transforms.py

```diff
@@ -32,7 +32,7 @@ def __init__(
         self.sep_token = sep_token
         self.max_seq_len = max_seq_len
 
-        self.token_transform = transforms.SentencePieceTokenizer(spm_model_path)
+        self.tokenizer = transforms.SentencePieceTokenizer(spm_model_path)
```

Comment:
Technically speaking, I think, this is BC-breaking change. The `dict_state` dumped from the previous version will stop working. Even if the BC-breaking is not an issue, as a good practice I recommend to split name change into a different PR, considering the possibility such as cherry-picking at minor release or something.

Reply:
Thanks for raising this up. I will keep change it back to original and do the naming change in separate PR.